lib/instrumentation/modules/memcached.js

```diff
@@ -13,17 +14,34 @@ module.exports = function (memcached, agent, { version, enabled }) {
     return memcached
   }
   agent.logger.debug('shimming memcached.prototype.command')
+  const [type, subtype] = ['db', 'memcached']
+
   shimmer.wrap(memcached.prototype, 'command', wrapCommand)
+  shimmer.wrap(memcached.prototype, 'connect', wrapConnect)
   return memcached
 
+  function wrapConnect (original) {
+    return function wrappedConnect () {
+      const activeSpan = agent._instrumentation.activeSpan
```

Comment:
I think what you really want here is `currentSpan` and not `activeSpan`.

Reply:
Come to think of it, that might actually explain why you have to do the "hack" down below

Reply:
Yeah, that totally did the job. Thanks for pointing it out.